packages/exhibitor/build

```diff
@@ -9,6 +9,8 @@ rm -rf "$PKG_PATH/usr/zookeeper/"{src,docs,contrib}
 
 mkdir -p "$PKG_PATH/bin"
 ln -s "$PKG_PATH/usr/zookeeper/bin/zkCli.sh" "$PKG_PATH/bin/zkCli.sh"
+# zkCli.sh expects zkEnv.sh to be in the same directory.
+ln -s "$PKG_PATH/usr/zookeeper/bin/zkEnv.sh" "$PKG_PATH/bin/zkEnv.sh"
```

Comment:
I see. Got it. 

I had already approved the PR and still had this doubt. Thanks for clarifying.